raiden-dapp/src/views/Channels.vue

```diff
@@ -55,18 +55,31 @@
 </template>
 
 <script lang="ts">
-import { Component, Vue } from 'vue-property-decorator';
+import { Component, Mixins } from 'vue-property-decorator';
 import { mapGetters } from 'vuex';
 import { ChannelState, RaidenChannel } from 'raiden-ts';
 import ChannelList from '@/components/ChannelList.vue';
 import ListHeader from '@/components/ListHeader.vue';
+import { Token } from '@/model/types';
+import AddressUtils from '@/utils/address-utils';
+import NavigationMixin from '@/mixins/navigation-mixin';
 
 @Component({
   components: { ListHeader, ChannelList },
-  computed: mapGetters(['channels'])
+  computed: {
+    ...mapGetters(['channels']),
+    ...mapGetters({
+      getToken: 'token'
```

Comment:
No need to change the pattern here, as it's just nitpick, or in all other components you applied it, but you really don't need to `map*` everything. If your `$store.getters.token`  is used just once (in the token computed/getter, afaics), you'd do better of going directly accessing `$store` there instead of polluting the class namespace with something used only in a single place and easily accessible directly from there.

Reply:
Sure I will go and change it :)